stepped-config/src/main.js

```diff
@@ -0,0 +1,137 @@
+var cc = DataStudioApp.createCommunityConnector();
+
+// https://developers.google.com/datastudio/connector/reference#isadminuser
+function isAdminUser() {
+  return false;
+}
+
+function optionsForState(state) {
+  switch (state) {
+    case 'IL': {
+      return [['Chicago', 'CHI'], ['Springfield', 'SPI']];
```

Comment:
nit: Rather than an array, does it make sense to return these as an object/key-value pairs? This should also get rid of the magic array index in getconfig.

Reply:
Good suggesting, switched over.